packages/components/package.json

```diff
@@ -23,7 +23,7 @@
     "build": "npm run build:clean && npm run build:js && npm run build:styles && npm run build:types && rm -rf dist/playground",
     "build:clean": "rm -rf dist/",
     "build:js": "babel src -d dist --extensions '.js,.jsx,.ts,.tsx' --ignore src/**/*.spec.ts,src/**/*.spec.tsx --ignore src/**/*.stories.ts,src/**/*.stories.tsx",
-    "build:styles": "postcss src/**/*.css --dir dist/ --base src/",
+    "build:styles": "postcss \"src/**/*([a-zA-Z-_])?(.module).css\" --dir dist/ --base src/ --verbose",
```

Comment:
prevents `stories.module.css` files from getting put in dist.

Reply:
Would it be a lot of work to move the ButtonGrid out to a separate place now? Inspired by Brad Frost's repo he shared, something like:
```
.storybook
  - components/ButtonGrid
      - ButtonGrid.module.css
      - ButtonGrid.tsx
      - ButtonGrid.stories.tsx
```

I don't love the idea of having files that are only for storybook in `src/`

Reply:
that's a good point - I'll explore it

Reply:
I'll move `tokens.stories` out as well